rapidpro/models/common.py

```diff
@@ -0,0 +1,16 @@
+from rapidpro.utils import generate_new_uuid
+
+
+class Exit:
+    def __init__(self, destination_uuid=None, exit_uuid=None):
+        self.uuid = exit_uuid if exit_uuid else generate_new_uuid()
+        self.destination_uuid = destination_uuid
```

Comment:
I feel like Exit should contain the next node rather than just its uuid, to make graph traversal easier.